manifests/t/Tencent/WeType/0.9.0.69/Tencent.WeType.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Tencent.WeType
+PackageVersion: 0.9.0.69
+Installers:
+- InstallerUrl: https://wetype.wxqcloud.qq.com/app/win/WeTypeSetup_0.9.0.69.exe
+  Architecture: x86
```

Comment:
```suggestion
  Architecture: x64
```

Reply:
Hi @StarsBySea,

`x86` packages are validated on a 32-bit OS, so I'm guessing this is a 64-bit application.

Reply:
Thank you. You're right!